server/command/daily_summary.go

```diff
@@ -29,11 +29,11 @@ func (c *Command) dailySummary(parameters ...string) (string, bool, error) {
 		}
 		return postStr, false, nil
 	case "tomorrow":
-		_, err := c.MSCalendar.GetDaySummaryForUser(time.Now().Add(time.Hour*24), c.user())
+		postStr, err := c.MSCalendar.GetDaySummaryForUser(time.Now().Add((time.Hour*24)*4), c.user())
```

Comment:
Why do we add `(time.Hour*24)*4` here? Also do we need to take into account weekends (maybe that's the intention here)?

Reply:
Oh no no no, I accidentaly commited that. Used to get the output of a specific day for a screenshot.

Reply:
Though this begs the question, should we have a special case for weekends? Currently, the daily summary job avoids sending the summary on the weekend, so there is already some convention in the plugin to avoid weekends. Maybe the `tomorrow` command should do something similar?

Reply:
Didn't check that code, is it configurable? Does it take into consideration countries were weekends are longer/shorter/on different days?

Reply:
Here is the line that does this https://github.com/mattermost/mattermost-plugin-mscalendar/blob/f226b5e7b2620c1ab625a2f2e4a31cbe6c1a6b30/server/mscalendar/daily_summary.go#L270

It's currently not configurable, but we can use `WorkingHours` to figure out the correct days if available for gcal:

https://github.com/mattermost/mattermost-plugin-mscalendar/blob/65b2e6734e76a6384a71101d809377f8b104aa62/server/remote/user.go#L13-L25